### ISSUE TITLE
Knobs on "Filter by price" do not have the correct screen-reader text

### ISSUE THREAD

**Issue by opr** [woocommerce/woocommerce-gutenberg-products-block#4378]
## Describe the bug
When using the "Filter by price" block, if you click the individual knobs with a screen reader active, it reads out the value without the decimal place. £200.00 is read out as "twenty thousand".

https://www.loom.com/share/3f32bcd3b3ee4e5c9c7d19ac3447e4d3

Seems semi-related to #1657 

## To reproduce

1. Add the "Filter by price" block to a page
2. Activate VoiceOver or some other screen reader.
3. Interact with the knobs on the slider and listen to the readout, it is not correct.

## Expected behavior
The screen reader text should be correct.
